tests/test_api.py

```diff
@@ -43,3 +44,36 @@ def test_available_broken_dataset(private_and_public_repository):
     df = audb.available(only_latest=True)
     assert len(df) > 0
     assert "broken-dataset" not in df
+
+
+def test_versions(tmpdir, repository):
```

Comment:
Is the test covering the line when `except audbackend.BackendError` is raised?
If yes, I have difficulty understanding why.-

Reply:
The test covers the line as the `repository` we add to `audb.config.REPOSITORIES` does not exist. When we call `audb.versions(...)` it goes through all repositories and calls `backend = utils.access_backend(repository)` for them (line 608 in `audb/core/api.py`). If a host or repository does not exist, a `audbackend.BackendError` is raised, compare https://github.com/audeering/audbackend/blob/84b723c70bd92ddb4f05a97dc974397d07131d1c/audbackend/core/api.py#L87 and https://github.com/audeering/audbackend/blob/84b723c70bd92ddb4f05a97dc974397d07131d1c/audbackend/core/utils.py#L26-L32

Reply:
I added a comment in the code, to explain when `audbackend.BackendError` is raised:

```python
        except audbackend.BackendError:
            # If the backend cannot be accessed,
            # e.g. host or repository do not exist,
            # we skip it
            # and continue with the next repository
            continue
```
and I introduced a `non_existing_repository` variable in the test, to make it easier to understand.

Reply:
Ok great then it steps over the  `audbackend.BackendError` altogether, and as it loops over `config.REPOSITORIES` anyhow the line will be  covered. Happy with that. Resolving.